aws_eden_core/methods.py

```diff
@@ -315,19 +304,19 @@ def config_add_env(bucket_name: str, key: str, env_name: str, env_cname: str, en
     return True
 
 
-def config_delete_env(bucket_name: str, key: str, env_name: str, env_cname: str, update_key: str):
+def endpoints_delete_env(bucket_name: str, key: str, name: str, fqdn: str, update_key: str):
```

Comment:
Spelling inconsistencies is here:

```suggestion
def endpoints_delete(bucket_name: str, key: str, name: str, fqdn: str, update_key: str):
```